command/ripe_atlas/traceroute.go

```diff
@@ -0,0 +1,167 @@
+package ripe_atlas
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/innogames/slack-bot/v2/bot"
+	"github.com/innogames/slack-bot/v2/bot/matcher"
+	"github.com/innogames/slack-bot/v2/bot/msg"
+	"github.com/innogames/slack-bot/v2/client"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/slack-go/slack"
+	"io"
+	"net/http"
+	"net/netip"
+	"time"
+)
+
+type tracerouteCommand struct {
+	bot.BaseCommand
+	cfg Config
+}
+
+func (c *tracerouteCommand) GetMatcher() matcher.Matcher {
+	return matcher.NewGroupMatcher(
+		matcher.NewRegexpMatcher(`traceroute (?P<TGT>.*)`, c.traceroute),
+	)
+}
+
+func (c *tracerouteCommand) traceroute(match matcher.Result, message msg.Message) {
+	destination := match.GetString("TGT")
+
+	c.AddReaction(":stopwatch:", message)
+	defer c.RemoveReaction(":stopwatch:", message)
+
+	var af int
+	address, err := netip.ParseAddr(destination)
+	if err != nil {
```

Comment:
You could simplify the `if` by adding an `|| address.Is6()` here. BTW why are we defaulting to IPv6 when the destination could not be parsed? Is this an implicit "is a hostname" check?

Reply:
This is more of a: if it can't be parsed, it's a hostname and if it's a hostname, use IPv6. The AF defined here is later used for the `MeasurementRequest`.

I need to actually test what would happen in the case a hostname has IPv4 only.

Reply:
As an idea we could also launch probes for both families